site/docs/v1/tech/apis/jwt.adoc

```diff
@@ -308,7 +311,11 @@ include::docs/src/json/jwt/refresh-post-response.json[]
 
 [.api]
 [field]#access_token# [type]#[String]#::
-The encoded access token. This cookie is written in the response as an HTTP Only session cookie.
+The encoded access token. This cookie is written in the response as an `HTTPOnly`, `SameSite=Lax` session cookie.
+
+[field]#refresh_token# [type]#[String]#::
+The encoded refresh token. This cookie is written in the response as an `HTTPOnly`, `SameSite=Lax` session cookie.
```

Comment:
The `SameSite` is default to `Lax` but it can be modified via configuration. See `fusionauth-app.cookie-same-site-policy`.

Reply:
Great, thanks. I'll omit it, then, to keep things simple.